kafka-connect-aws-s3/src/test/scala/io/lenses/streamreactor/connect/aws/s3/source/config/ReadTextModeTestFormatSelection.scala

```diff
@@ -32,7 +34,20 @@ class ReadTextModeTestFormatSelection extends AnyFlatSpec with Matchers {
           S3PropsKeyEnum.ReadEndTag.entryName   -> "</p>",
         ),
       ),
-    ) should be(Some(StartEndTagReadTextMode("<p>", "</p>")))
+    ) should be(Some(StartEndTagReadTextMode("<p>", "</p>", 1024)))
```

Comment:
could the 1024 be a variable with a name or comment that explains the magic number?